meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -297,21 +297,29 @@ PREFERRED_PROVIDER_u-boot-fw-utils_mx8mm = "libubootenv"
 PREFERRED_RPROVIDER_u-boot-fw-utils_mx8mm = "libubootenv"
 SOTA_CLIENT_FEATURES_append_mx8mm = " ubootenv"
 UBOOT_DTB_LOADADDRESS_mx8mm = "0x43000000"
+PREFERRED_PROVIDER_virtual/trusted-firmware-a_mx8mm ?= "imx-atf"
+EXTRA_IMAGEDEPENDS_append_mx8mm = "virtual/trusted-firmware-a"
+UBOOT_CLASSES_sota_mx8mm = "uboot-fitimage"
+ATF_BINARY_mx8mm = "arm-trusted-firmware.bin"
+UBOOT_CONFIG_mx8mm = ""
+UBOOT_SUFFIX_mx8mm = "bin"
 IMAGE_BOOT_FILES_mx8mm = "imx-boot ${KERNEL_IMAGETYPE} ${@make_dtb_boot_files(d)} boot.scr uEnv.txt"
-IMAGE_BOOT_FILES_sota_mx8mm = "imx-boot boot.itb"
+IMAGE_BOOT_FILES_sota_mx8mm = "imx-boot u-boot.itb boot.itb"
 WKS_FILE_DEPENDS_append_mx8mm = " u-boot-default-script"
 PREFERRED_PROVIDER_u-boot-default-script_mx8mm = "u-boot-base-scr"
 PREFERRED_PROVIDER_u-boot-default-script_sota_mx8mm = "u-boot-ostree-scr-fit"
 KERNEL_IMAGETYPE_sota_mx8mm = "fitImage"
 KERNEL_CLASSES_sota_mx8mm = " kernel-lmp-fitimage "
-OSTREE_KERNEL_ARGS_imx8mmevk ?= "console=tty1 console=ttymxc1,115200 earlycon=ec_imx6q,0x30890000,115200 root=/dev/mmcblk2p2 rootfstype=ext4"
-PREFERRED_PROVIDER_virtual/dtb_imx8mmevk ?= "lmp-device-tree"
 ## iMX8: Use latest NXP BSP downstream kernel
 PREFERRED_PROVIDER_virtual/kernel_mx8mm ?= "linux-lmp-fslc-imx"
 MACHINE_FIRMWARE_mx8mm = "linux-firmware-imx-sdma-imx7d"
-WKS_FILE_mx8mm_sota = "sdimage-imx8-sota.wks.in"
+WKS_FILE_mx8mm_sota = "sdimage-imx8-spl-sota.wks.in"
+IMXBOOT_TARGETS_sota_mx8mm = "flash_evk_spl"
```

Comment:
`s/mx8mm/imx8mmevk/g` ?

Reply:
I was debating whether this would be too generic.  We only have mfgtools defined for `imx8mmevk` atm.  But I would imagine those will be copied / pasted for the next target.

Other than mfgtools, what would be the downside of defining this as the proper boot path for all `mx8mm` ?